LayoutTests/TestExpectations

```diff
@@ -6041,3 +6041,7 @@ http/tests/loading/oauth.html [ DumpJSConsoleLogInStdErr ]
 http/tests/security/contentSecurityPolicy/video-with-https-url-allowed-by-csp-media-src-star.html [ DumpJSConsoleLogInStdErr ]
 imported/w3c/web-platform-tests/clipboard-apis/feature-policy/clipboard-write/clipboard-write-disabled-by-feature-policy.tentative.https.sub.html [ DumpJSConsoleLogInStdErr ]
 imported/w3c/web-platform-tests/html/infrastructure/urls/terminology-0/document-base-url-initiated-grand-parent.https.window.html [ DumpJSConsoleLogInStdErr ]
+
+# Screen orientation
+## We don't implment test_driver.minimize_window()
```

Comment:
Is `window.internals.setPageVisibility` good enough to implement test_driver.minimize_window?
If so, could we try implementing it this way?

Reply:
I had a look. We might be able to use the same semantics ((ab)using page visibility), but I don't think we can use `window.internal`, as it needs to be done through `TestDriver`. 

Is my understanding correct?

Reply:
Can we update  LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js to use window.internals?

Reply:
Indeed we can, I recently discovered. 🎉 I’ll patch something up.